File: C99/1100-1199/1160-1169/1165.c
#include <stdio.h>

int main()
{
    int num, n, cont;
    scanf("%d", &num);
    for (int j = 0; j < num; j++)
    {
        scanf("%d", &n);
        cont = 0;
        for (int i = 1; i <= n; i++)
        {
            if (n % i == 0)
            {
                cont++;
            }
        }
        if (cont == 2)
        {
            printf("%d eh primo\n", n);
        }
        else
        {
            printf("%d nao eh primo\n", n);
        }
    }
    return 0;
}

File: python/1100-1199/1160-1169/1165.py
num = int(input())
for n in range(num):
    entry = int(input())
    count = 0
    for i in range(1, entry + 1):
        if entry % i == 0:
            count += 1
    if count == 2:
        print(f"{entry} eh primo")
    else:
        print(f"{entry} nao eh primo")